client/src/app/components/edit-profile/edit-profile.component.html

```diff
@@ -0,0 +1,55 @@
+<div class="wrapper">
```

Comment:
change this class name to container as it fit with the conventions we use in other pages